Makefile

```diff
@@ -11,7 +11,7 @@ YARN := yarn
 BOWER := bower
 
 # Options to pass to pulp when building
-BUILD_OPTIONS := -- --stash --censor-lib --strict
```

Comment:
These options are valid, intentional, and important to me (they ase psa options). They aren't to be removed.